tasks/test-tutorial

```diff
@@ -16,34 +16,46 @@ yarn install
 
 if [ -z "$1" ]
   then
-  echo "You have not supplied a path to a Redwood project."
+  echo "You have not supplied a path to a RedwoodJS project."
   echo "We will create one for you."
   echo "We will copy './packages/create-redwood-app/template' and the packages/*"
-
   TMP_DIR=$(mktemp -d -t redwood)
-
-  cd ../../packages/create-redwood-app
-  yarn babel-node src/create-redwood-app.js $TMP_DIR  --no-yarn-install
 else
   echo "You have supplied a path $1, we will not create a new "
   echo "Redwood project, we will use the app you have specified."
   TMP_DIR=$1
 fi
 
-if [ -z "$1" ]
+if [[ -z "$1"  || "$CREATE_RWJS_PROJECT" == "1" ]]
+  then
+  echo "Creating new RedwoodJS Project..."
+  cd ../../packages/create-redwood-app
+  yarn babel-node src/create-redwood-app.js $TMP_DIR  --no-yarn-install
+fi
+
+if [[ -z "$1"  || "$CREATE_RWJS_PROJECT" == "1" ]]
   then
   cd "$SCRIPTPATH/../"
-  # build all the packages
+  # build all the packages, but not the TypeScript
+  # since it is slow and we do not need it.
   yarn build:clean && yarn lerna run build:js
 fi
 
 cd $TMP_DIR
 
-if [ -z "$1" ]
+if [[ -z "$1" || "$CREATE_RWJS_PROJECT" == "1" ]];
   then
-  # make the e2e tests use the packages from this repo.
+  echo "Linking packages from $SCRIPTPATH"
+  # make the e2e tests use the packages from this Redwood Framework Repo.
   ln -s "$SCRIPTPATH/../packages"
 fi
 
 yarn install
-yarn rw dev --fwd="--open=false" & cd "$SCRIPTPATH/e2e"; yarn cypress open --env RW_PATH=$TMP_DIR
+
+if [ -z "$CI" ]
+  then
+  # TODO: Figure out how to run these automatically without opening the cypress runner.
```

Comment:
You can do it using the cypress npm module, but you're going to have a bad time on Windows. I created this issue almost a year ago... https://github.com/cypress-io/cypress/issues/7367

Reply:
Cool, I actually realized that the runner is the best experience on my local computer. This script will die, but I'll remove this comment.